packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Colors, Indent, Options, Print, Plugin} from '../types.js';
+
+const escapeHTML = require('./lib/escapeHTML');
+const HTML_ELEMENT_REGEXP = /(HTML\w*?Element)/;
+const test = isHTMLElement;
+
+function isHTMLElement(value: any) {
+  const toStringed = value.toString();
```

Comment:
I'm a bit worried about performance here. This will call toString on any value passed to pretty-format. toString may be expensive if it is overwritten on a class instance. Can we do `Object.prototype.toString.call`?